net-p2p/createtorrent/createtorrent-1.1.4-r1.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="Create BitTorrent files easily"
+HOMEPAGE="http://www.createtorrent.com/"
+SRC_URI="http://www.createtorrent.com/${PN}-${PV}.tar.gz"
```

Comment:
`${P}.tar.gz`?

Reply:
This change is to permit me to use the same source URI used by 1.1.4 but change the revision number.  If I take over the source code on my own web page, then it would not be needed.